dotCMS/src/main/java/com/dotcms/util/pagination/TemplateView.java

```diff
@@ -0,0 +1,360 @@
+        this.isNew = builder.isNew;
+        this.showOnMenu = builder.showOnMenu;
+        this.drawedBody = builder.drawedBody;
+        this.countAddContainer = builder.countAddContainer;
+        this.countContainers = builder.countContainers;
+        this.headCode = builder.headCode;
+        this.theme = builder.theme;
+        this.themeName = builder.themeName;
+        this.footer = builder.footer;
+        this.friendlyName = builder.friendlyName;
+        this.header = builder.header;
+        this.modDate = builder.modDate;
+        this.modUser = builder.modUser;
+        this.owner = builder.owner;
+        this.name = builder.name;
+        this.title = builder.title;
+        this.sortOrder = builder.sortOrder;
+    }
+
+    public String getIdentifier() {
+        return identifier;
+    }
+
+    public String getInode() {
+        return inode;
+    }
+
+    public String getCategoryId() {
+        return categoryId;
+    }
+
+    public String getBody() {
+        return body;
+    }
+
+    public String getSelectedimage() {
+        return selectedimage;
+    }
+
+    public String getImage() {
+        return image;
+    }
+
+    public boolean isDrawed() {
+        return drawed;
+    }
+
+    public boolean isDeleted() {
+        return deleted;
+    }
+
+    public boolean isLive() {
+        return live;
+    }
+
+    public boolean isLocked() {
+        return locked;
+    }
+
+    public boolean isWorking() {
+        return working;
+    }
+
+    public boolean isNew() {
+        return isNew;
+    }
+
+    public boolean isShowOnMenu() {
+        return showOnMenu;
+    }
+
+    public String getDrawedBody() {
+        return drawedBody;
+    }
+
+    public int getCountAddContainer() {
+        return countAddContainer;
+    }
+
+    public int getCountContainers() {
+        return countContainers;
+    }
+
+    public String getHeadCode() {
+        return headCode;
+    }
+
+    public String getTheme() {
+        return theme;
+    }
+
+    public String getThemeName() {
+        return themeName;
+    }
+
+    public String getFooter() {
+        return footer;
+    }
+
+    public String getFriendlyName() {
+        return friendlyName;
+    }
+
+    public String getHeader() {
+        return header;
+    }
+
+    public Date getModDate() {
+        return modDate;
+    }
+
+    public String getModUser() {
+        return modUser;
+    }
+
+    public String getOwner() {
+        return owner;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public int getSortOrder() {
+        return sortOrder;
+    }
+
+    public static final class Builder {
+        private  String identifier;
+        private  String inode;
+        private  String categoryId;
+        private  String body;
+        private  String selectedimage;
+        private  String image;
+        private  boolean drawed;
```

Comment:
Codacy found an issue: [Field drawed has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6042990)